INSTALL-WINDOWS.md

```diff
@@ -1,89 +1,52 @@
-We recommend the use of Windows Subsystem for Linux (WSL) to install PyBaMM, see the
-instructions below to get PyBaMM working using Windows, WSL and VSCode.
+## Prerequisites
 
-## Install WSL
+To use and/or contribute to PyBaMM, you must have Python 3.6 or 3.7 installed (note that 3.8 is not yet supported).
 
-Follow the instructions from Microsoft
-[here](https://docs.microsoft.com/en-us/windows/wsl/install-win10). When given the
-option, choose the Ubuntu 18.04 LTS distribution to install. Don't forget to initialise
-the Ubuntu installation using the instructions given
-[here](https://docs.microsoft.com/en-us/windows/wsl/initialize-distro).
+To install Python 3 download the installation files from [Python's website](https://www.python.org/downloads/windows/).
```

Comment:
We could point to the [official python on Windows guide](https://docs.python.org/3.7/using/windows.html).
The most straightforward way would be to use the [full installer](https://docs.python.org/3.7/using/windows.html#windows-full).
Actually the command `python` is not automatically available, unless the `Add python 3.7 to PATH` option is ticked on the installer.
I think something that would always work is to use the [python launcher](https://docs.python.org/3.7/using/windows.html#python-launcher-for-windows) `py`.

So
```
py -m venv env
env\Scripts\activate.bat
pip install pybamm
```
I'm not sure if its worth putting in so much detail, we could maybe just point to the windows guide?

Reply:
I have now added a link to the guide, and a line reminding them to tick the box, so the command `python` should work. Let me know if I should still change it to `py` and I will fix it.

Reply:
BTW, feel free to merge the PR when you think it is fine, as I can't.